### DISCUSSION TITLE
[Feature Request] Add message component

### DISCUSSION THREAD

**Discussion by BuZhiHeWei** [heroui-inc/heroui#1894] [Feature requests]
### Is your feature request related to a problem? Please describe.

Display global messages as feedback in response to user operations.

### Describe the solution you'd like

1. To provide feedback such as success, warning, error etc.
2. A message is displayed at top and center and will be dismissed automatically, as a non-interrupting light-weighted prompt.

### Describe alternatives you've considered

I'm sorry, I have no suggestion.

**Comment by siddiquiaffan** [comment]
You might like `react-hot-toast` package.

A fully featured toast package. Which works really well with customisation, promises, etc.

**Reply by kanbykanby** [reply]
This will introduce a new package to the user's project, potentially affecting the overall bundle size. The message/notification component is both simple and useful. Why not consider adding it to this UI components library?

**Reply by kanbykanby** [reply]
That package is awesome by the way.

**Reply by siddiquiaffan** [reply]
Somehow I agree with you, but acc to me why  should we waste time when there's a great work has already been done to solve a particular issue.

And one more thing, what if I'm not using this UI library and I need that message/notif component?

**Comment by fac3m4n** [comment]
Try [sonner](https://sonner.emilkowal.ski)

**Comment by wingkwong** [comment]
see toast: #314
